js/uuid-helper.js

```diff
@@ -0,0 +1,5 @@
+const { v4: uuidv4 } = require('uuid');
+
+window.generateUUID = function() {
```

Comment:
I bet there is a clever way to avoid global scope but I don't want to increase scope.
Could we add that to the backlog so it can be prioritised later?

Reply:
I'm wondering if browserify is the right module bundler for our scope. I can add it to the backlog @pthanvi

Reply:
I will add a ticket to the backlog